src/syft/lib/python/range.py

```diff
@@ -0,0 +1,36 @@
+# stdlib
+from typing import Any
+from typing import Optional
+from typing import Union
+
+# syft relative
+from ...core.common import UID
+from ...proto.lib.python.range_pb2 import Range as Range_PB
+from .primitive_factory import PyPrimitive
+
+
+class Range(PyPrimitive):
+    def __init__(
+        self,
+        start: Any = None,
```

Comment:
Shouldn't this be int/Int?

Reply:
`mypy` had an issue with `int`, with `Any` it seems to be fine.